documentation/library_guide/dynamic_selection_api_main.rst

```diff
@@ -0,0 +1,92 @@
+Dynamic Selection API
+#####################
+
+The dynamic selection API is an experimental feature in the |onedpl_long| (|onedpl_short|).
+Dynamic selection provides functions for choosing a *resource* using a 
+*selection policy*.  By default, the resources selected via these APIs 
+in |onedpl_short| are SYCL queues. There are several functions and selection 
+policies provided as part of the API. 
+
+The selection policies include: ``fixed_resource_policy`` that always selects a 
+specified resource, ``round_robin_policy`` that rotates between resources,
+``dynamic_load_policy`` that chooses the resource that has the fewest outstanding 
+submissions, and ``auto_tune_policy`` that chooses the best resources based on runtime 
+profling information.  
```

Comment:
Please correct spelling to "profiling"

Reply:
done (and caught another spelling error of the same word in a different file)